Formula/fdroidserver.rb

```diff
@@ -15,25 +15,35 @@ class Fdroidserver < Formula
 
   depends_on :python3
   depends_on "freetype"
-  depends_on "jpeg"
```

Comment:
If you look in other formulae, our "standard" pillow resource block is heavily patched.